butterfree/load/writers/historical_feature_store_writer.py

```diff
@@ -140,9 +140,12 @@ def write(
         """
         partition_df = self._create_partitions(dataframe)
 
-        dataframe = self.check_schema(
-            spark_client, partition_df, feature_set.name, self.database
-        )
+        if self.debug_mode:
+            dataframe = partition_df
+        else:
+            dataframe = self.check_schema(
+                spark_client, partition_df, feature_set.name, self.database
+            )
 
         if self.interval_mode:
             if self.debug_mode:
```

Comment:
Good, I believe that we can remove the dynamic partition configuration to debug_mode :smile_cat: